platform/jn516x/dev/micromac-radio.c

```diff
@@ -57,14 +57,24 @@
 #include "JPT.h"
 #include "PeripheralRegs.h"
 
-void vMMAC_SetChannelAndPower(uint8 u8Channel, int8 i8power);
-
```

Comment:
then I would suggest to either fix this signature, or update the sdk that Travis uses